tests/performance/test_data_analysis/unit_tests/test_statistical_comparator.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""Unit tests for the StatisticalComparator class"""
+
+import pytest
+import pandas as pd
+
+from statistical_data_analyzer import StatisticalComparator
```

Comment:
```suggestion
import pandas as pd
import pytest

from statistical_data_analyzer import StatisticalComparator
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3